css/nav_pagination.css

```diff
@@ -80,11 +80,11 @@
   flex-wrap: wrap;
 }
 
-.nav_pagination_current__page {
+.nav_pagination__current--page {
   background-color: #0083C3;
 }
 
-.nav_pagination_list a.nav_pagination_current__page {
+.nav_pagination__list a.nav_pagination__current--page {
```

Comment:
I think when you use BEM, you should use the class name. Please, check here because you still have an anchor.

Reply:
Thank you so much for this feedback, we forgot to take it out. We will fix it.